src/hound_writer.rs

```diff
@@ -0,0 +1,201 @@
+//! A wrapper for the hound library that writes binauralized audio to
+//! the user-speciifed output file.
+
+use crate::component::{Component, ComponentError};
+use hound::{Error as HoundError, WavSpec, WavWriter};
+
+use std::fs::File;
+use std::io::BufWriter;
+
+
+/// A monitor wrapper for the hound WavWriter that writes out binauralized
+/// audio
+pub struct HoundWriter {
+    writer: Option<WavWriter<BufWriter<File>>>,
+}
+
+impl HoundWriter {
+    /// Instantiates a new HoundWriter, which wraps the hound WavWriter
+    fn new(file: &'static str, wave_spec: WavSpec) -> Self {
```

Comment:
```suggestion
    pub fn new(file: impl AsRef<Path>, wave_spec: WavSpec) -> Self {
```

Reply:
This way we can send a `String`, or an `&str`, or an `&'static str` or anything that can be turned into a `Path` into this function.